saleor/migrate/tasks.py

```diff
@@ -0,0 +1,98 @@
+import dj_database_url
+
+from faker import Factory
+
+from celery import Task, signals
+from ..celeryconf import app
```

Comment:
Let's organize the imports like the following: 
```py
import foobar
import foobiz

from fred import 
from waldo import

from ...quux import
from ...qux import
from ..bar import
from ..biz import
from .corge import
from .garply import
```

That seems to be how the team from saleor organizes the imports.

Reply:
And sort the three groups separately.